extensions/pick-your-wallpaper/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pick-your-wallpaper",
+  "title": "Pick Your Wallpaper",
+  "description": "Lets you pick a wallpaper from a folder of finder",
+  "icon": "command-icon.png",
+  "author": "hugomarfani",
+  "categories": [
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "choose-wallpaper",
+      "title": "Choose Wallpaper",
+      "description": "Pick a wallpaper from a folder of finder",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "wallpaperFolder",
+      "title": "Wallpaper Folder Directory",
+      "description": "The directory where your wallpapers are stored",
+      "type": "directory",
+      "required": true
+    },
+    {
+      "name": "showTitle",
+      "label": "Show Wallpaper Titles",
+      "title": "Shows Wallpaper Titles",
+      "description": "Shows the wallpaper titles in the picker",
+      "type": "checkbox",
+      "defaultValue": true,
+      "required": false
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.50.4",
+    "run-applescript": "^6.1.0"
+  },
+  "devDependencies": {
+    "@raycast/eslint-config": "1.0.5",
+    "@types/node": "18.8.3",
+    "@types/react": "18.0.9",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
+    "eslint": "^7.32.0",
+    "eslint-plugin-react": "^7.32.2",
+    "prettier": "^2.5.1",
+    "typescript": "^4.4.3"
+  },
+  "scripts": {
+    "build": "ray build -e dist",
+    "dev": "ray develop",
+    "fix-lint": "ray lint --fix",
```

Comment:
You might want to run this in order to fix syntax styling issues and pass the checks :)